app/models/amusement_park.rb

```diff
@@ -1,3 +1,9 @@
 class AmusementPark < ApplicationRecord
   has_many :rides
+  has_many :ride_mechanics, through: :rides
+  has_many :mechanics, through: :ride_mechanics
```

Comment:
Nice work adding these additional associations! For the future, you can also just skip right to mechanics like so:
`has_many :mechanis, through: :rides`